app/src/main/java/com/codesoom/assignment/domain/Task.java

```diff
@@ -0,0 +1,106 @@
+package com.codesoom.assignment.domain;
+
+import com.codesoom.assignment.common.Builder;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+/**
+ *  할일 도메인을 개념적으로 표현한 엔티티입니다.
+ */
+public class Task {
+    private String taskId;
+    private String title;
+    private LocalDateTime createdAt;
+    private LocalDateTime updatedAt;
+    private Status status;
+
+    /**
+     * 할일 도메인의 상태를 그룹화한 enum입니다.
+     */
+    private enum Status {
+        REMOVE, INCOMPLETE, COMPLETION
+    }
+
+    /**
+     * 전달 받은 아이디와 제목을 초기화 시킵니다.
+     *
+     * @param taskBuilder
+     */
```

Comment:
이렇게 하면 다른 필드들이 추가될 때마다 주석도 계속 바뀌겠죠. 이렇게 해보세요.
```suggestion
    /**
     * 할 일을 생성합니다.
     *
     * @param taskBuilder
     */
```